game/test/ecs/items/ItemActiveTest.java

```diff
@@ -0,0 +1,53 @@
+package ecs.items;
+
+import static org.junit.Assert.*;
+
+import ecs.components.InventoryComponent;
+import ecs.entities.Entity;
+import org.junit.Test;
+import org.mockito.Mockito;
+
+public class ItemActiveTest {
+
+    /** Tests if set callback is called. */
+    @Test
+    public void testUseCallback() {
+        IItemUse callback = Mockito.mock(IItemUse.class);
+        ItemActive item =
+                Mockito.mock(
+                        ItemActive.class,
+                        Mockito.withSettings()
+                                .useConstructor("name", "description", callback)
+                                .defaultAnswer(Mockito.CALLS_REAL_METHODS));
+        Entity entity = new Entity();
+        item.use(entity);
+        Mockito.verify(callback).onUse(entity, item);
+    }
+
+    /** Tests if no exception is thrown when callback is null. */
+    @Test
+    public void testUseNullCallback() {
+        ItemActive item =
+                Mockito.mock(
+                        ItemActive.class,
+                        Mockito.withSettings()
+                                .useConstructor("name", "description", null)
+                                .defaultAnswer(Mockito.CALLS_REAL_METHODS));
+        Entity entity = new Entity();
+        item.use(entity);
+    }
+
+    /** Tests if item is removed from inventory after use. */
+    @Test
+    public void testItemRemovedAfterUseWithDefaultCallback() {
+        ItemActive item = new ItemActive() {};
+        Entity entity = new Entity();
+        InventoryComponent inventoryComponent = new InventoryComponent(entity, 2);
+        inventoryComponent.addItem(item);
+        assertTrue(inventoryComponent.getItems().contains(item));
```

Comment:
assertTrue oder assertFalse am besten immer mit einer Message versehen damit man einfacher versteht warum hier etwas true oder false sein sollte.
```suggestion
        assertTrue("ActiveItem needs to be addedd to the Inventory.",inventoryComponent.getItems().contains(item));
```